node-definitions/Microsoft/win server 2019.yaml

```diff
@@ -0,0 +1,48 @@
+id: win2019
```

Comment:
Thanks for doing this.  Can you include a README for this node def that explains how to go about getting/building an image def for it?  Also, can you lowercase "microsoft" to be consistent with other vendor names here.